openmdao/recorders/recording_manager.py

```diff
@@ -22,7 +24,12 @@ def __init__(self):
         init.
         """
         self._recorders = []
-        self.rank = 0
+        self._has_serial_recorders = False
```

Comment:
Ken, sorry some of the early parallel recording code was mixed in with what was pushed up. It will be used !